libs/output-mapping/tests/DeferredTasks/LoadTableQueueTest.php

```diff
@@ -136,7 +136,57 @@ public function testWaitForAllWithError(): void
 
         try {
             $loadQueue->waitForAll();
-            self::fail('waitForAll shoud fail with InvalidOutputException-');
+            self::fail('WaitForAll shoud fail with InvalidOutputException.');
+        } catch (InvalidOutputException $e) {
+            self::assertSame(
+                'Failed to load table "myTable": Table with displayName "test" already exists.',
+                $e->getMessage()
+            );
+        }
+
+        $tablesResult = $loadQueue->getTableResult();
+
+        $tables = $tablesResult->getTables();
+        self::assertCount(0, iterator_to_array($tables));
+
+        $tablesMetrics = $tablesResult->getMetrics()->getTableMetrics();
+        self::assertCount(0, iterator_to_array($tablesMetrics));
+    }
+
+    public function testWaitForAllWithSapiUserErrorOnMetadataApplyThrowsInvalidOutputException(): void
```

Comment:
přijde mi, že je to úplně stejný test jako testWaitForAllWithErrorThrowsInvalidOutputException

Reply:
Ocenuju ze ctes testy:)

Reply:
ten mock je tady https://github.com/keboola/platform-libraries/pull/41/files#diff-fc8876d3dd840028947e8027086dc5d1ea48f86ab700e64f08b203016fb2a590R278

Reply:
testy jsem upravil tak davam znovu na review

Reply:
no podle mě je jeden z těch testů zbytečnej, ale jak myslíš

Reply:
Podlě mě není. `waitForAll` dělá 2 věci:

- čeká než doběhne job v Storage API a pokud skončí errorem
- u success jobu se snaží nastavit metadata tabulce

A ty potřebuješ otestovat že to hodí `InvalidOutputException` když job skončí errorem a když job skončí jako success, ale metadata failnou.